awscli/clidocs.py

```diff
@@ -716,3 +730,62 @@ def _line_has_tag(self, line):
 
     def doc_subitems_start(self, help_command, **kwargs):
         pass
+
+
+class GlobalsRenderer:
```

Comment:
For the name of this class, let's see if we can figure out a different name than `Render`. Mainly that name is generally preserved for rendering generated docs to the shell that the user is using (e.g. the renderer classes in the `help.py` module). How about `GlobalOptionsDocumenter`? I'm open for ideas

Reply:
I figured this was an imprecise name. I'm not too opinionated on this, just want to make sure it's consistent with the rest of the codebase. I was also thinking `GlobalOptionsGenerator`. Is `Generator` reserved for anything?